explanation/jaas_security_scope.rst

```diff
@@ -20,6 +20,8 @@ We recommend the `Canonical identity platform`_ as the preferred IdP for JAAS. T
 authentication on behalf of JAAS using OAuth 2.0 and OIDC. For authorisation, JAAS provides this by means
 of tags and ReBAC (Relation-Based Access Control).
 
+See the following pages for more details on how JAAS provides :doc:`authentication <./authentication>` and authorization.
```

Comment:
Depending on which PR lands first, the other should change this line.

Reply:
What spelling do we use btw? I used `authorization` but I see you use `authorisation`.